maxim/logger/writer.py

```
@@ -367,7 +367,9 @@ def flush_log_files(self):
                     self.maxim_api.push_logs(self.config.repository_id, logs)
                     os.remove(os.path.join(self.logs_dir, file))
                 except Exception as e:
-                    scribe().warn(f"[MaximSDK] Failed to access filesystem. Error: {e}")
+                    scribe().warning(
+                        f"[MaximSDK] Failed to access filesystem. Error: {e}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)